packages/lib/services/commands/ToolbarButtonUtils.ts

```diff
@@ -49,13 +59,14 @@ export default class ToolbarButtonUtils {
 
 		const command = this.service.commandByName(commandName, { runtimeMustBeRegistered: true });
 
-		const output = {
+		const output: ToolbarButtonInfo = {
+			type: 'button',
 			name: commandName,
 			tooltip: this.service.label(commandName),
 			iconName: command.declaration.iconName,
 			enabled: newEnabled,
 			onClick: async () => {
-				void this.service.execute(commandName);
+				await this.service.execute(commandName);
 				void focusEditorIfEditorCommand(commandName, this.service);
```

Comment:
Changing this `void` to an `await` fixes a regression. Without this change, clicking "attach" doesn't dismiss the keyboard on iOS.